_osmtogtfs/gtfs_dummy.py

```diff
@@ -18,69 +36,122 @@ def _create_dummy_calendar():
              'friday': 1, 'saturday': 0, 'sunday': 0, 'start_date': 20170101, 'end_date': 20190101}]
 
 
-def _create_dummy_trips_and_stoptimes(processor, calendar):
+def _create_dummy_trips(routes, stops, route_stops, calendar):
     trips = []
-    stoptimes = []
-    for route_id, route in processor.routes.items():
-        if len(processor.route_stops.get(route_id, [])) < 2:
+
+    for route_id, route in routes.items():
+
+        if len(route_stops.get(route_id, [])) < 2:
             continue
+
         for cal in calendar:
-        # For the sake of simplicity, we assume a fixed number of trips per service day.
-        # Even though in reality there are less number of trips on weekends and holidays.
-        # We assume trips begin from 5:00 AM and run untill 11:00 PM and there is one trip
-        # every 20 minutes. Therefore in total we add 54 trips per route per service day.
-        # 18 service hours per day * 3 trips per hour = 54
+            # For the sake of simplicity, we assume a fixed number of trips per service day.
+            # Even though in reality there are less number of trips on weekends and holidays.
+            # We assume trips begin from 5:00 AM and run untill 11:00 PM and there is one trip
+            # every 20 minutes. Therefore in total we add 54 trips per route per service day.
+            # 18 service hours per day * 3 trips per hour = 54
             for idx in range(54):
-                trip_id = '{route_id}{sequence}'.format(route_id=route_id, sequence=idx+1)
+
+                trip_id = \
+                    '{route_id}.{service_id}{sequence}'.format(route_id=route_id,
+                        service_id=cal['service_id'],
```

Comment:
@mehdisadeghi could you make the service_id here a number as well? I dont mean to change it globally (necessarily) but here. That way I can read the trip_id in kamal as a double (example: 1234.557). Otherwise I would be force to store all the strings which is always very heavy :(

Reply:
@carocad Do you need the value of `service_id`? If not I can simply set it to some integer.

Reply:
One more thing. What if the part after `.` begins with `0`? Will it cause problems for you?

Reply:
> @carocad Do you need the value of service_id? If not I can simply set it to some integer.

No I dont.

> One more thing. What if the part after . begins with 0? Will it cause problems for you?

I am assuming that is a 0 (zero) not the vowel O.  If it is a zero then that is not a problem, as long as left and right of the dot are numbers everything is fine.

Reply:
@carocad that's just what I wanted. I'll send a patch soon.